src/Ticket.php

```diff
@@ -2120,6 +2120,9 @@ public function post_addItem()
             if (isset($this->fields["status"]) && ($this->fields["status"] == self::SOLVED)) {
                 $type = "solved";
             }
+            if (isset($this->fields["status"]) && ($this->fields["status"] == self::CLOSED)) {
+                $type = "closed";
+            }
             NotificationEvent::raiseEvent($type, $this);
```

Comment:
Following logic should permit to send both notifications. 

```suggestion
            NotificationEvent::raiseEvent("new", $this);
            
            if (isset($this->fields["status"]) && ($this->fields["status"] == self::SOLVED)) {
                NotificationEvent::raiseEvent("solved", $this);
            }
            if (isset($this->fields["status"]) && ($this->fields["status"] == self::CLOSED)) {
                NotificationEvent::raiseEvent("closed", $this);
            }
```